shaders/CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ endif()
 # ==================================================================================================
 add_library(${TARGET} STATIC ${RESGEN_SOURCE})
 target_include_directories(${TARGET} PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+set_target_properties(${TARGET} PROPERTIES FOLDER Filament/Generated)
```

Comment:
Those shaders are hand written and shouldn't go in Generated. They should go in Filament/Shaders

Reply:
Done!